src/jingle_sip/mod_jingle_sip_backend.erl

```diff
@@ -0,0 +1,199 @@
+-module(mod_jingle_sip_backend).
+
+-include("mongoose.hrl").
+
+-type call_id() :: binary().
+-type incoming_request() :: binary().
+-type outgoing_handle() :: binary().
+
+-export([init/2]).
+-export([set_incoming_request/5]).
+-export([set_incoming_handle/2]).
+-export([set_outgoing_request/4]).
+-export([set_outgoing_handle/4]).
+-export([set_outgoing_accepted/1]).
+-export([set_incoming_accepted/1]).
+-export([get_incoming_request/1]).
+-export([get_outgoing_handle/1]).
+-export([get_session_info/1]).
+-export([remove_session/1]).
+
+-record(jingle_sip_session, {sid,
+                             dialog,
+                             state,
+                             direction,
+                             request,
+                             node,
+                             meta}).
+
+init(_Host, _Opts) ->
+    mnesia:create_table(jingle_sip_session,
+                        [{ram_copies, [node()]},
+                         {attributes, record_info(fields, jingle_sip_session)}]).
```

Comment:
add_local_copy?

Reply:
Good question, I'm quite sure it's not needed. I'll double check.

Reply:
During the test, when node2 joined the `mnesia:info/0` showed:
```running db nodes   = [mongooseim@localhost,ejabberd2@localhost]
stopped db nodes   = []
master node tables = []
remote             = []
ram_copies         = [acl,anonymous,bosh_session,config,external_component,
                      external_component_global,iq_response,
                      jingle_sip_session,local_config,mod_register_ip,node,
                      reg_users_counter,route,s2s,session,sm_session]```
Looks like the `jingle_sip_session` is in local ram copies.